### PR TITLE
Natron sdk linux fixes

### PR DESCRIPTION
**Pull request by fonkle**
Additional fixes for the Natron SDK Linux build script.

Some src URL changes and the (temporary) removal of Qt 5.6 and Pyside2.
The script still stops at mariadb, in need of a restart.

p.s. Sorry for the new PR but I couldn't find a way to reopen the merged one.

### CODE REVIEW

File: tools/jenkins/include/scripts/build-Linux-sdk.sh [hunk 1]
```diff
@@ -1270,7 +1270,10 @@ if [ ! -s "$SDK_HOME/osmesa/lib/pkgconfig/gl.pc" ] || [ "$(env PKG_CONFIG_PATH=$
         download "$OSMESA_LLVM_SITE" "$OSMESA_LLVM_TAR"
         mkdir -p "${SDK_HOME}/llvm" || true
     fi
-    git clone https://github.com/devernay/osmesa-install
+    
+    #git clone https://github.com/devernay/osmesa-install
+    git clone https://github.com/fonkle/osmesa-install
```
Comment (devernay) [inline tools/jenkins/include/scripts/build-Linux-sdk.sh]: oops?
please send a PR to my repo if there are any non-merged changes, and revert to the main repo

File: tools/jenkins/include/scripts/build-Linux-sdk.sh [hunk 2]
```diff
@@ -2945,117 +2949,117 @@ if [ ! -s "$SDK_HOME/bin/gdb" ]; then
...
-        # Copy private headers to includes because Pyside2 needs them
-        ALL_MODULES_INCLUDE=("$QT5PREFIX"/include/*)
-        for m in "${ALL_MODULES_INCLUDE[@]}"; do
-            (cd "$m"; ln -sfv "$QT5_VERSION"/*/private .)
-        done
-
-        end_build "${module}-${QT5_VERSION}"
-    done
-fi
+# if [ "${REBUILD_QT5:-}" = "1" ]; then
```
Comment (devernay) [inline tools/jenkins/include/scripts/build-Linux-sdk.sh]: please enclose this portion within a `if false; then` ... `fi` block, without reindenting. This helps keeping track of file history

File: tools/jenkins/include/scripts/build-Linux-sdk.sh [hunk 3]
```diff
@@ -3546,7 +3550,7 @@ fi
 echo
 echo "Check for broken libraries and binaries... (everything is OK if nothing is printed)"
 # SDK_HOME=/opt/Natron-sdk; LD_LIBRARY_PATH=$SDK_HOME/qt4/lib:$SDK_HOME/gcc/lib:$SDK_HOME/lib:$SDK_HOME/qt5/lib
-for subdir in . ffmpeg-gpl2 ffmpeg-lgpl libraw-gpl2 libraw-lgpl qt4 qt5; do
+for subdir in . ffmpeg-gpl2 ffmpeg-lgpl libraw-gpl2 libraw-lgpl qt4; do # qt5; do
```
Comment (devernay) [inline tools/jenkins/include/scripts/build-Linux-sdk.sh]: no need to remove qt5 from this line, it doesn't hurt

**Comment by fonkle** [comment]
I have closed this PR to put the non-merged changes into a new branch, for which I will send a new PR upstream.

File: tools/jenkins/include/scripts/build-Linux-sdk.sh (hunk 3 above)
Comment (fonkle) [reply tools/jenkins/include/scripts/build-Linux-sdk.sh]: Actually it does error out when missing the qt5 lib.

**Comment by devernay** [comment]
Ok

**Pull request closed without merging**